include/aikido/distance/JointAvoidanceConfigurationRanker.hpp

```diff
@@ -20,7 +20,20 @@ class JointAvoidanceConfigurationRanker : public ConfigurationRanker
       statespace::dart::ConstMetaSkeletonStateSpacePtr metaSkeletonStateSpace,
       ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton);
 
+  /// Constructor
+  ///
+  /// \param[in] metaSkeletonStateSpace Statespace of the skeleton.
+  /// \param[in] metaSkeleton Metaskeleton of the robot.
+  /// \param[in] weights Weights over joints to compute distance.
+  JointAvoidanceConfigurationRanker(
+      statespace::dart::ConstMetaSkeletonStateSpacePtr metaSkeletonStateSpace,
+      ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton,
+      std::vector<double> weights);
+
 protected:
+  /// Set limits appropriately to account for infinite limits.
```

Comment:
Does this only work for infinite-limit joints?

Reply:
If there are infinite limit joints, we would like them to be ignored for distance computation but the ranker works for finite/infinite limits.